### ISSUE TITLE
User Manual?

### ISSUE THREAD

**Issue by computersarecool** [rncbc/qjackctl#79]
Hi,

I am wondering if there is a user manual for this program?

My specific questions are:
- Why do my MIDI devices show up in the ALSA tab (not the MIDI tab) of the "Connect" button and
- Why use the patchbay instead of the "Connect" button

but I have other questions which I think could be easier answered by reading a manual.

**Comment by rncbc** [comment]
this (deang old page) might help in both accounts:
https://www.rncbc.org/drupal/node/76

cheers

**Comment by computersarecool** [comment]
Yes this is pretty helpful - where is the right place to ask follow up questions?
Do you want help making a proper manual from this?

there are so many options in JACK I don't see how people could know how to use qJackctrl without a manual.
